page/running_tournament.js

```diff
@@ -1,14 +1,82 @@
 function page (req, res) {
```

Comment:
Calling this page without an id results in:`Error [ERR_HTTP_HEADERS_SENT]: Cannot render headers after they are sent to the client`